cumulusci/core/config/__init__.py

```diff
@@ -35,22 +35,20 @@ class TaskConfig(BaseConfig):
     pass
 
 
-from cumulusci.core.config.BaseTaskFlowConfig import BaseTaskFlowConfig
-
+from cumulusci.core.config.base_task_flow_config import BaseTaskFlowConfig
 
 # inherit from BaseTaskFlowConfig
 from cumulusci.core.config.project_config import BaseProjectConfig
 
+# inherit from SfdxOrgConfig
+from cumulusci.core.config.scratch_org_config import ScratchOrgConfig
+
```

Comment:
Why did you move this one? I guess as long as there aren't import cycle problems it's fine, but ScratchOrgConfig extends SfdxOrgConfig so it made sense to me for them to be listed in the previous order.